docs/jazzy.yml

```diff
@@ -11,6 +11,14 @@ skip_undocumented: Yes
 hide_documentation_coverage: Yes
 
 custom_categories:
+  - name: Examples
+    children:
+      - Storyboards
+      - Advanced
+      - Basic
```

Comment:
Rearrange these examples in order of difficulty, so that “Basic” comes before “Advanced”.